specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/SourceControls.json

```diff
@@ -171,6 +171,13 @@
               "$ref": "#/definitions/SourceControl"
             }
           },
+          {
+            "name": "x-ms-github-auxiliary",
+            "in": "header",
+            "description": "Github personal access token used for SourceControl.",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Is this API version on this PUT API already open to public and code is receiving this header ? 
And this is just to update the swagger ?

Reply:
In backend code, from day one we will require the header. Here is to fix the issue in the swasgger of new api version.